### PR TITLE
Release prep for v0.16.0

### PR DESCRIPTION
**Pull request by dvdplm**
Release preparation for v0.16.0

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -24,15 +24,15 @@ chameleon = "0.1.0"
 scale-info = { version = "1.0.0", features = ["bit-vec"] }
 futures = "0.3.13"
 hex = "0.4.3"
-jsonrpsee = { version = "0.7.0", features = ["macros", "ws-client", "http-client", "client-ws-transport"] }
+jsonrpsee = { version = "0.8.0", features = ["macros", "ws-client", "http-client", "client-ws-transport"] }
```
Comment (niklasad1) [inline Cargo.toml:27]:
```suggestion
jsonrpsee = { version = "0.8.0", features = ["ws-client", "http-client"] }
```

Comment (niklasad1) [reply Cargo.toml:27]: The `feature macros` is unused and `client-ws-transport` is enabled by the `ws-client`.

**Review by ascjones — APPROVED** [review]
:tada:

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,73 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.16.0] - 2022-02-01
+
```
Comment (jsdw) [inline CHANGELOG.md:10]: I wonder whether it's worth a brief message at the top like the following?

```
*Note*: This is a significant release which introduces support for V14 metadata and macro based codegen, as well as making many breaking changes to the API.
```

**Review by jsdw — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (dvdplm) [reply CHANGELOG.md:10]: Good pt, adding.

**Pull request merged**